Makefile

```diff
@@ -3,20 +3,22 @@
 #
 all: binary
 
+PKGS := $(shell go list ./... | grep -vE '/vendor/|github.com/docker/cli/man|github.com/docker/cli/docs/yaml$$')
```

Comment:
I don't think we need to exclude `man/` or `docs/` from unit testing anymore , now that we have all the dependencies installed in `vendor/`.